packages/docusaurus-theme-classic/src/theme/Navbar/MobileSidebar/Header/index.tsx

```diff
@@ -16,6 +16,7 @@ function CloseButton() {
   return (
     <button
       type="button"
+      aria-label="Close navigation bar"
```

Comment:
We usually translate all our aria labels by default

You can use the Docusaurus `translate()` API and then update the default theme translations with `yarn workspace @docusaurus/theme-translations update` (so that translators can find and edit the untranslated strings)